src/test/java/teammates/test/pageobjects/InstructorFeedbackEditPage.java

```diff
@@ -569,6 +570,16 @@ public void selectQuestionNumber(int qnNumber, int newQnNumber) {
         selectDropdownByVisibleValue(qnNumSelect, String.valueOf(newQnNumber));
     }
 
+    public boolean isSelectQuestionNumberEnabled(int qnNumber) {
+        WebElement qnNumSelect = browser.driver.findElement(By.id("questionnum-" + qnNumber));
```

Comment:
```
public int getSelectedQuestionNumber(int qnNumber) {
    Select qnNumSelect = new Select(getSelectQuestionNumberDropdown(qnNumber));
    return Integer.parseInt(qnNumSelect.getFirstSelectedOption().getText());
}
```
The main point of this is so that if the identifier of the question number dropdown changes, you do not have to look through all the methods to find the correct places to change, so the code is more maintainable.